libs/sdk-core/src/breez_services.rs

```diff
@@ -453,6 +453,19 @@ impl BreezServices {
         Ok(())
     }
 
+    pub async fn prepare_withdraw(
+        &self,
+        prepare_withdraw_request: PrepareWithdrawRequest,
```

Comment:
```suggestion
        req: PrepareWithdrawRequest,
```

Easier to just name the field `req`.